### PR TITLE
Radiotimes scores reviews

### PR DESCRIPTION
**Pull request by atish-metabroadcast**
Please see https://jira.metabroadcast.com/browse/MBST-14273

### CODE REVIEW

File: src/main/java/org/atlasapi/remotesite/rt/RtFilmProcessor.java [hunk 1]
```diff
@@ -160,7 +166,29 @@ public void process(Element filmElement) {
         if (hasValue(ukCinemaCertificate)) {
             film.setCertificates(certificate(ukCinemaCertificate));
         }
+
+        ArrayList<Review> reviews = new ArrayList(2);
```
Comment (Spartakovic) [inline src/main/java/org/atlasapi/remotesite/rt/RtFilmProcessor.java]: doesn't it warn about absence of brackets?

**Comment by Spartakovic** [comment]
lgtm

File: src/main/java/org/atlasapi/remotesite/rt/RtFilmProcessor.java [hunk 2]
```diff
@@ -160,7 +166,31 @@ public void process(Element filmElement) {
         if (hasValue(ukCinemaCertificate)) {
             film.setCertificates(certificate(ukCinemaCertificate));
         }
-        
+
+        ArrayList<Review> reviews = new ArrayList(2);
+        Element normalReview = filmElement.getFirstChildElement("normal_review");
+        if (hasValue(normalReview)) {
```
Comment (tfm) [inline src/main/java/org/atlasapi/remotesite/rt/RtFilmProcessor.java]: Perhaps
```
Optional<Review> review1 = extractReview("normal_review");
if (review1.isPresent()) {
  reviews.add(review1.get())
}
```

File: src/main/java/org/atlasapi/remotesite/rt/RtFilmProcessor.java [hunk 3]
```diff
@@ -160,7 +166,31 @@ public void process(Element filmElement) {
...
+        Element shortReview = filmElement.getFirstChildElement("short_review");
+        if (hasValue(shortReview)) {
+            reviews.add(new Review(Locale.ENGLISH, shortReview.getValue()));
+        }
+
+        film.setReviews(reviews);
+
+        Element starRating = filmElement.getFirstChildElement("rating");
+        if (hasValue(starRating)) {
+            int ratingValue = Character.getNumericValue(starRating.getValue().charAt(0));
```
Comment (tfm) [inline src/main/java/org/atlasapi/remotesite/rt/RtFilmProcessor.java]: `Ints.tryParse()` returns a null rather than <0

**Comment by tfm** [comment]
Very minor stylistic things, otherwise 🚀

File: src/main/resources/config/environment.properties [hunk 4]
```diff
@@ -7,7 +7,7 @@ local.host.name=localhost
 
 #Mongo host/db configuration.
 mongo.host=127.0.0.1
-//TODO switch this to atlas before you commit it.
+#TODO switch this to atlas before you commit it.
```
Comment (Dalamar42) [inline src/main/resources/config/environment.properties]: You can remove this comment. It's irrelevant. Puppet takes care of having the correct production values here. It's not even `atlas` anymore, it's `atlas-split`

File: src/main/resources/config/environment.properties [hunk 5]
```diff
@@ -45,7 +45,7 @@ updaters.bbc-audience-data.enabled=false
 updaters.itv.enabled=false
 updaters.c4.enabled=false
 updaters.c4pmlsd.enabled=false
-updaters.five.enabled=true
+updaters.five.enabled=false
```
Comment (Dalamar42) [inline src/main/resources/config/environment.properties]: Please confirm puppet is setting a value for this before committing this to master

File: src/main/java/org/atlasapi/remotesite/rt/RtFilmProcessor.java (hunk 3 above)
Comment (Dalamar42) [inline src/main/java/org/atlasapi/remotesite/rt/RtFilmProcessor.java]: Similarly to @tfm 's comment above you could abstract the parsing logic away and just return Optional<Integer>

**Pull request merged**